src/Administration/Resources/app/administration/src/app/component/form/sw-text-editor/sw-text-editor-toolbar/sw-text-editor-toolbar.scss

```diff
@@ -1,80 +0,0 @@
-@import "~scss/variables";
```

Comment:
Now you've deleted the whole editor toolbar styles, probably not what you wanted.
please bring this file back and squash your commits to one relevant commit.

Reply:
That's what i wanted. This file was related to my other PR https://github.com/shopware/platform/pull/633

Reply:
yep but that file should not be part of this PR, if we would merge this as is, all the editor toolbar styles would be deleted, which would probably break the editor and additionally your PR #633 would have merge conflicts as both PRs touch the same file.

So please split the code changes between your PRs so we can merge them indepently.

Reply:
of course .. my fault .. Should be fixed now.

Reply:
Could please squash your commits to one, with a descriptive commit message?
This [guide](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing) should help you if aren't familiar with sqashing commits.

Keep in mind that you need to force push after squashing commits

Reply:
I have to dive deeper into git ... :) 
Thanks for your hint. Should be hopefully fixed now.

Reply:
@keulinho Sorry tor the late reply. I squashed the commits now into one.